extension/index.html

```diff
@@ -12,7 +12,5 @@
 
 <body>
     <H1>HIMANSHU 007</H1>
-    <h2>DELETE THIS LINE</h2>
-

```

Comment:
insert this line here
<p>i am getting good hold of github</p>

Reply:
resolve issue.